src/routes/MainRoutes.js

```diff
@@ -90,7 +90,7 @@ const MainRoutes = {
     },
     {
       path: 'activities',
-      element: <ProUserRoute />,
+      // element: <ProUserRoute />,
       children: [{ path: '', element: <Activities /> }]
```

Comment:
You can remove the children, and directly use <Activities /> in element

```suggestion
      element: <Activities />
```

Reply:
done